src/scripts/opm_status.js

```diff
@@ -0,0 +1,51 @@
+//  Description:
+//    get opm status from api
+//
+//  Dependencies:
+//    None
+//
+//  Commands:
+//    @bot opm status
+//
+//  Author:
+//    lauraggit
+
+const { directMention } = require("@slack/bolt");
+const axios = require("axios");
+
+// :greenlight: :redlight: :yellowlight:
+const icons = {
+  Open: ":greenlight:",
+  Alert: ":yellowlight:",
+  Closed: ":redlight:",
+};
+
+module.exports = (robot) => {
+  robot.message(
+    directMention(),
+    /opm status/i,
+    async ({ event: { thread_ts: thread }, say }) => {
+      try {
+        const { data, status } = await axios.get(
+          "https://www.opm.gov/json/operatingstatus.json"
+        );
+        if (status !== 200) {
+          throw new Error("Invalid status");
+        }
+        say({
+          icon_emoji: icons[data.Icon],
+          text: `${data.StatusSummary} for ${data.AppliesTo}. (<${data.Url}|Read more>)`,
+          thread_ts: thread,
+          unfurl_links: false,
+          unfurl_media: false,
+        });
+      } catch (e) {
```

Comment:
I'm legit not sure what errors this will catch. This is another old script and I just bonked it until it fit into the new API without thinking too much about it. 😂